src/cmd-buildextend-live

```diff
@@ -215,11 +215,13 @@ def generate_iso():
     tmpisofile = os.path.join(tmpdir, iso_name)
 
     img_metal_obj = buildmeta.get_artifact_meta("metal", unmerged=True)["images"].get("metal")
-    img_metal4k_obj = buildmeta.get_artifact_meta("metal4k", unmerged=True)["images"].get("metal4k")
-    if img_metal_obj is None or img_metal4k_obj is None:
-        raise Exception("Live image generation requires `metal` and `metal4k` images")
+    if img_metal_obj is None:
+        raise Exception("Live image generation requires missing `metal` image")
```

Comment:
nit: `missing` seems ambiguous to me.  This could be saying that it requires the `metal` image to be absent.  :upside_down_face:

Reply:
Heh, fair. Fixed!